src/ReflectionClosure.php

```diff
@@ -910,6 +915,11 @@ protected function fetchItems()
                             $name .= $token[1];
                             $alias = $token[1];
                             break;
+                        case T_NAME_QUALIFIED:
```

Comment:
There are more tokens than this to consider. ;)

Reply:
This one fixes one specific test that I wrote that was passing on 7.4. It's 23:00 here so I'm heading to bed for today and will continue tomorrow

Reply:
This is still unresolved.

Reply:
Can you be more specific?